lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -145,7 +145,7 @@ PortInfo calyx::getPortInfo(BlockArgument arg) {
 
 /// Returns whether the given operation has a control region.
 static bool hasControlRegion(Operation *op) {
-  return isa<ControlOp, SeqOp, IfOp, WhileOp, ParOp, StaticRepeatOp,
+  return isa<ControlOp, SeqOp, IfOp, RepeatOp, WhileOp, ParOp, StaticRepeatOp,
```

Comment:
Not something to change in this PR, but these `isa` checks should probably check if op is ControlLike.